### ISSUE TITLE
Is there a way to increase the horizontal drag buffer?

### ISSUE THREAD

**Issue by mmccooyyy** [angular-ui-tree/angular-ui-tree#966]
*Note*: for support questions, please use our [Gitter channel](https://gitter.im/angular-ui-tree/angular-ui-tree). This repository's issues are reserved for feature requests and bug reports.

**Do you want to request a *feature* or report a *bug*?**
Feature: add an option to set the horizontal drag buffer to a set distance.

**What is the current behavior?**
The horizontal drag buffer is very small, making it difficult to drop a node under the correct parent when there are multiple levels.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar.**

**What is the expected behavior?**
I would expect there to be an option to change this value or to copy the value from the left-margin value of the node.

**What is the motivation / use case for changing the behavior?**
It's difficult to drop a node to a specific parent that is not the last child or first parent.

**Which versions of Angular, UI Tree and which browser / OS are affected by this issue? Did this work in previous versions of Angular, UI Tree? Please also test with the latest version.**
Angular 1.3, Chrome 60, Windows 10

**Other information (e.g. stacktraces, related issues, suggestions how to fix)**

Video: https://youtu.be/i5K5rm1S_Iw

**Comment by chasonhuang** [comment]
hello!  Have you solved this problem?   if you have, please tell me the way.  thanks!

**Comment by mmccooyyy** [comment]
@chasonhuang yes I was.  I duplicated the changes from [this pull request](https://github.com/angular-ui-tree/angular-ui-tree/pull/934) and that fixed it.  I found that a buffer of 15-25 is what worked best for me.  Not sure why the dev hasn't merged that pull request yet.  @Voles any plan to merge #934?
